Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Tower/FanSpeed/Controller.mo

```diff
@@ -0,0 +1,371 @@
+                       group="Less coupled plant", enable=not closeCoupledPlant));
+  parameter Real speChe=0.005 "Lower threshold value to check fan or pump speed"
+    annotation (Dialog(tab="Advanced"));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput chiLoa[nChi](
+    final unit=fill("W", nChi),
+    final quantity=fill("Power", nChi)) if hasWSE "Current load of each chiller"
+    annotation (Placement(transformation(extent={{-140,120},{-100,160}}),
+      iconTransformation(extent={{-140,170},{-100,210}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uChi[nChi]
+    "Chiller enabling status: true=ON"
+    annotation (Placement(transformation(extent={{-140,100},{-100,140}}),
+      iconTransformation(extent={{-140,140},{-100,180}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uWSE if hasWSE
+    "Waterside economizer enabling status: true=ON"
+    annotation (Placement(transformation(extent={{-140,80},{-100,120}}),
+      iconTransformation(extent={{-140,110},{-100,150}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uTowSpe(
+     final min=0,
+     final max=1,
+     final unit="1") "Tower fan speed"
+     annotation (Placement(transformation(extent={{-140,60},{-100,100}}),
+       iconTransformation(extent={{-140,80},{-100,120}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TChiWatSup(
+    final unit="K",
+    final quantity="ThermodynamicTemperature") if hasWSE
+    "Chilled water supply temperature"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+      iconTransformation(extent={{-140,50},{-100,90}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TChiWatSupSet(
+    final unit="K",
+    final quantity="ThermodynamicTemperature")
+    "Chilled water supply temperature setpoint"
+    annotation (Placement(transformation(extent={{-140,20},{-100,60}}),
+      iconTransformation(extent={{-140,20},{-100,60}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput reqPlaCap(
+    final unit="W",
+    final quantity="Power") "Current required plant capacity"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+      iconTransformation(extent={{-140,-10},{-100,30}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uMaxTowSpeSet[nChi](
+    final min=fill(0, nChi),
+    final max=fill(1, nChi),
+    final unit=fill("1", nChi))
+    "Maximum cooling tower speed setpoint from each chiller head pressure control loop"
+    annotation (Placement(transformation(extent={{-140,-40},{-100,0}}),
+      iconTransformation(extent={{-140,-40},{-100,0}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uTowSta[nTowCel]
+    "Cooling tower cell operating status: true=running tower cell"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
+      iconTransformation(extent={{-140,-70},{-100,-30}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uPla
+    "Plant enabling status"
```

Comment:
Don't we always use `Sta` for any eanble/disable status (unless it is in the name of the model)?